tests/acceptance/conftest.py

```diff
@@ -57,10 +57,5 @@ def pytest_configure(config):
     )
 
     subprocess.call(
-        ["yarn", "webpack"],
-        env={
-            "NODE_ENV": "development",
-            "PATH": os.environ["PATH"],
-            "NODE_OPTIONS": "--max-old-space-size=4096",
```

Comment:
Can we restore the `--max-old-space-size` here

Reply:
Sure but I'd like to understand why. After this patch `yarn webpack` should stay under the limit. Seems like it is mostly Jest and other stuff pushing it over.

I'll make this change and merge after that so we can remove this again later if we determine it is not needed as I suggested.

Reply:
It looks like I added this at https://github.com/getsentry/sentry/pull/15688 

At the time, I had some issues running some acceptance tests locally. This is because the frontend may get rebuilt prior to running  acceptance tests; and this procedure is based on the existence of `.webpack.meta`, and if the frontend was built over an hour ago.

I think it might be a good idea to leave this here for now. We can definitely revisit this once the TypeScript conversion reaches over 90% of the codebase (right now it's ~47%).